config/application.rb

```diff
@@ -9,6 +9,7 @@
 require 'action_mailer/railtie'
 require 'sprockets/railtie'
 require 'rails/test_unit/railtie'
+# require 'rails/all'
```

Comment:
Let's add a note here about why we comment out `rails all`, so I have to stop looking it up over and over, haha